packages/dynamite/dynamite/build.yaml

```diff
@@ -1,8 +1,15 @@
 builders:
-  dynamite:
-    import: 'package:dynamite/builder.dart'
```

Comment:
If the `dynamite` builder is removed then why do the other build.yaml files switch to `dynamite|dynamite_openapi`?

Reply:
Ah ok :+1: